packages/hdwallet-keepkey/src/ethereum.ts

```diff
@@ -21,8 +21,8 @@ export function ethSupportsNativeShapeShift(): boolean {
   return true;
 }
 
-export function ethSupportsEIP1559(): boolean {
-  return this.ethSupportsEIP1559();
+export async function ethSupportsEIP1559(): Promise<boolean> {
```

Comment:
This breaks the way all of the "Supports" functions work. This is why we did firmware checks in axiom instead of hdwallet.

I don't like the idea of having some "Supports" functions be async and some be sync.